modules/navier_stokes/doc/content/source/variables/BernoulliPressureVariable.md

```diff
@@ -29,11 +29,28 @@ dimension explicit inlet boundary conditions related to velocity, one explicit
 inlet boundary condition related to temperature, and one explicit outlet
 boundary condition related to pressure. Thus physics-based discretizations of
 Euler flows typically (at least partially) upwinds information being advected by
-the flow field. However, given the progagation of pressure information upstream
+the flow field. However, given the propagation of pressure information upstream
 from the explicit outlet pressure boundary condition, we believe it reasonable
 to do the same for computing the Bernoulli pressure jump at porosity
 discontinuities.
 
+!alert! note title=Behavior with parallel execution
+
+In certain cases multiple porosity jump faces may be connected by several cells in a
+chain. For example at corners or when porous medium zones are one-cell wide. In such
+scenarios, the two-term expansion in the downstream cell for the determination of the
+face pressure can lead to a very long chain of calls, which can potentially lead to
+attempts of attempts of accessing cell values which are neither owned
+nor ghosted on the given rank. For this reason, the default value of
+[!param](/Variables/BernoulliPressureVariable/allow_two_term_expansion_on_bernoulli_faces)
+is `false`. If the user wants to enable two-term expansion for the pressure
+computation on the porosity ump faces, special attention should be paid to
+moving the porosity jump faces sufficiently far from each other or adding
```

Comment:
This isnt prescriptive enough. Do we need 2 cells in between each jump ?

Reply:
Well, unless we need skewness correction. Then we need more 3, added this detail.